SMDS_3/test/SMDS_3/test_c3t3_with_features.cpp

```diff
@@ -346,7 +346,7 @@ struct Tester
     // Test vertex iterators
     //-------------------------------------------------------
     std::cout << "Test vertex iterators\n";
-    const Vertex_handle& vertex_to_modify = c3t3.vertices_in_complex_begin();
+    Vertex_handle vertex_to_modify = c3t3.vertices_in_complex_begin();
```

Comment:
can you please remove the comment below? `vertex_to_modify` is not a dangling reference anymore with this change, it's just a `Vertex_handle`